homeassistant/components/renson/__init__.py

```diff
@@ -56,3 +63,66 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+def setup_hass_services(hass: HomeAssistant, renson_api: RensonVentilation) -> None:
+    """Set up the Renson platforms."""
+
+    async def set_timer_level(call: ServiceCall) -> None:
+        """Set timer level."""
+        level_string = call.data.get("timer_level", "Level1")
+        time = call.data.get("time", 0)
```

Comment:
```suggestion
        level_string = call.data["timer_level"]
        time = call.data.get["time"]
```

Schema enforces that the values are present.